scripts/remove

```diff
@@ -49,6 +49,7 @@ ynh_remove_systemd_config
 ynh_script_progression --message="Removing dependencies..."
 
 # Remove metapackage and its dependencies
+pip3 uninstall -y  merkletools PySocks coincurve maxminddb bencode
```

Comment:
Not sure it's a good idea to remove them if they are use by other apps or YunoHost